lib/salesforceiq.js

```diff
@@ -447,6 +448,196 @@ var SalesforceIQ = (function() {
     makeRequest('lists/' + listId, req, cb);
   };
 
+  /**
+   * Format fields for list item to match SalesforceIQ API specs,
+   * using ids pulled from the associated list
+   * (c.f. https://api.salesforceiq.com/#/curl#documentation_lists_list-items_list-item-object)
+   *
+   * @param {string}                  listId  Reference id for the list containing these fields
+   * @param {Object.<string, string>} data    Simple key/value store of field data
+   * @param {Function}                cb      Callback returning error info and formatted fields
+   */
+  SalesforceIQ.prototype.formatListFields = function(listId, data, cb) {
+    this.getList(listId, function(err, list) {
+      if (err) {
+        // REF console.log(err);
+        cb(err);
```

Comment:
This was flagged as an error with ESLint and I learned something new by reading about it!

http://eslint.org/docs/rules/callback-return